Project_team_1_HW2/Project_team_1_HW2.md

```diff
@@ -0,0 +1,35 @@
+
+## 2. What does it do or what kind of problem does it solve?
+
+The problem our project will solve is with the vast number of API options out there people don’t know which one to choose and spend more time searching through an array of APIs than actually reading the information they want. It's similar to the issue with Netflix in that you spend more time searching for content than actually looking at the content you want. Our project hopes to put all the popular APIs in one place so that people don’t need to spend forever searching; they can find the right information straight away or at least be pointed in the right direction so they spend a lot less time searching for what they want.  We have also included RSS feeds within our project so that people can be updated when a new article is posted so that they remain up to date with the latest content. We have also taken into consideration the different countries people come from which solves the issue of people struggling to find news relevant to them. Through research it has been found that most APIs tend to cover news from only the USA/UK so we have included sites that allow you to specify your country to provide relevant information solving the issue of having to search through hundreds if not thousands of results showing potentially irrelevant information. Users will have the choice of which county to display where possible which gives them the chance to read news from their own country and others to stay up to date. Overall our project uses research and our own experience to find the most relevant APIS over a range of popular categories and displays results in one place to make learning about technology easy and accessible for users. 
+
+To sum up, from the end-user's perspective, the situation our app is targeting is that of the multitude of news which is available to people in this day and age. Much like the above-mentioned APIs and the Netflix case, given the amount of news available on a daily basis to readers, it could be quite a daunting task to scour through countless websites to find news related to topics of interest, and this approach could turn out to be dispersive and time-consuming. By aggregating news content into a single up-to-date source and consolidating it into news bits displayed to the end user based on their preferences in content, our app hopes to simplify this task and "meet the user halfway" by proposing ready-to-access content they might be interested in.
+
+## 3. What are the key features of your system?
```

Comment:
2/3 While writing key features, Ideally we don't need to mention the technology used behind it.

Example:

Avoid:
- SQL database feature for user information - to keep user passwords, IDs, etc.

Do:
- Security that allow user to login and keep user password and ids